platform-core/cassandra-connector/src/main/java/org/ekstep/cassandra/store/CassandraStore.java

```diff
@@ -32,6 +33,8 @@
  */
 public abstract class CassandraStore {
 
+	protected ObjectMapper mapper = new ObjectMapper();
```

Comment:
Why we have added this declaration. Not used any where.

Reply:
It is used in child class . HierarchyStore